tests/staveconnector_tests.ts

```diff
@@ -19,7 +19,6 @@ const StaveConnectorTests = {
     const run = VexFlowTests.runTests;
     test('VF.* API', this.VF_Prefix);
     run('Single Draw Test', this.drawSingle);
-    run('Single Draw Test, 1px Stave Line Thickness', this.drawSingle1pxBarlines);
```

Comment:
It seems to be the only test changing the Stave Line Thickness. So I suggest leaving it changing `1px` to `2px`.

Reply:
Sure, I could save the default value, change it to something different, then restore the old default value at the end of the test case. Having a hard-coded 2 vs 1 will break again if we ever decide to change the default in the future.

Reply:
Savig the default value would be better, you are right.